### PR TITLE
Update the first page of the Clustering section

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1747.surge.sh

Updated pages:
- [Clustering](https://neo4j-docs-operations-1747.surge.sh/operations-manual/5/clustering)

File: modules/ROOT/pages/clustering/index.adoc [hunk 1]
```diff
@@ -9,23 +9,27 @@ This chapter describes the following:
...
 ** xref:clustering/monitoring/endpoints.adoc[Monitor cluster endpoints for status information] -- The endpoints and semantics of endpoints used to monitor the health of the cluster.
 * xref:clustering/disaster-recovery.adoc[Disaster recovery] -- How to recover a cluster in the event of a disaster.
 * xref:clustering/settings.adoc[Settings reference] -- A summary of the most important cluster settings.
+* xref:clustering/server-syntax.adoc[Server commands reference] -- Reference of Cypher administrative commands to add and manage servers.
 * xref:clustering/clustering-advanced/index.adoc[Advanced clustering] -- Some more advanced features of Neo4j clusters.
+** xref:clustering/clustering-advanced/default-database.adoc[Default database in a cluster] -- The initial default database created when the DBMS starts for the first time.
 ** xref:clustering/clustering-advanced/multi-data-center-routing.adoc[Multi-data center routing] -- Clusters on mutli-data centers.
+** xref:clustering/clustering-advanced/reconciler.adoc[Reconciler] -- An internal component which observes the requested state of a server and makes changes to the server to match that state.
```
Comment (renetapopova) [inline modules/ROOT/pages/clustering/index.adoc:28]:
```suggestion
** xref:clustering/clustering-advanced/reconciler.adoc[Reconciler] -- An internal component that observes the requested state of a server and makes changes to the server to match that state. 
```

File: modules/ROOT/pages/clustering/index.adoc [hunk 2]
```diff
@@ -9,23 +9,27 @@ This chapter describes the following:
...
 * xref:clustering/disaster-recovery.adoc[Disaster recovery] -- How to recover a cluster in the event of a disaster.
 * xref:clustering/settings.adoc[Settings reference] -- A summary of the most important cluster settings.
+* xref:clustering/server-syntax.adoc[Server commands reference] -- Reference of Cypher administrative commands to add and manage servers.
 * xref:clustering/clustering-advanced/index.adoc[Advanced clustering] -- Some more advanced features of Neo4j clusters.
+** xref:clustering/clustering-advanced/default-database.adoc[Default database in a cluster] -- The initial default database created when the DBMS starts for the first time.
 ** xref:clustering/clustering-advanced/multi-data-center-routing.adoc[Multi-data center routing] -- Clusters on mutli-data centers.
+** xref:clustering/clustering-advanced/reconciler.adoc[Reconciler] -- An internal component which observes the requested state of a server and makes changes to the server to match that state. 
 * xref:clustering/glossary.adoc[Clustering glossary] -- A glossary of terms used in the clustering documentation.
```
Comment (renetapopova) [inline modules/ROOT/pages/clustering/index.adoc:29]:
```suggestion
* xref:clustering/glossary.adoc[Clustering glossary] -- A glossary of terms related to the Neo4j clustering.
```

**Review by renetapopova — CHANGES_REQUESTED** [review]
Thanks, @NataliaIvakina! As I wrote here as a comment, I think we need to align the titles in the navigation and on the pages and reflect that here as well. Otherwise, the clustering section is inconsistent with the rest of the manual and also within itself. What do you think?

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1747.surge.sh

Updated pages:
- [Clustering](https://neo4j-docs-operations-1747.surge.sh/operations-manual/5/clustering)
- [Leadership, routing, and load balancing](https://neo4j-docs-operations-1747.surge.sh/operations-manual/5/clustering/setup/routing)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1747.surge.sh

Updated pages:
- [Clustering](https://neo4j-docs-operations-1747.surge.sh/operations-manual/5/clustering)
- [Leadership, routing, and load balancing](https://neo4j-docs-operations-1747.surge.sh/operations-manual/5/clustering/setup/routing)

**Review by renetapopova — APPROVED** [review]
Looks good now. Thanks!

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1747.surge.sh

Updated pages:
- [Advanced clustering](https://neo4j-docs-operations-1747.surge.sh/operations-manual/5/clustering/clustering-advanced)
- [Clustering](https://neo4j-docs-operations-1747.surge.sh/operations-manual/5/clustering)
- [Leadership, routing, and load balancing](https://neo4j-docs-operations-1747.surge.sh/operations-manual/5/clustering/setup/routing)

**Comment by NataliaIvakina** [comment]
we can't remove the _index.adoc_ file from the Advanced clustering section, because it's tied up to the Advanced causal clustering section in 4.4

**Pull request merged**